styles.css

```diff
@@ -93,7 +101,338 @@ ul.navbar-items {
   line-height: 1.5rem;
 }
 
-@media only screen and (max-width: 375px) {
+/* Works section styling */
+
+#works-section {
+  margin: 10px 0;
+  overflow-y: hidden;
+}
+
+.box-container {
+  display: flex;
+  margin: 50px 200px 50px 200px;
+}
+
+.left-section {
+  flex: 1;
+}
+
+.left-section img {
+  float: left;
+  margin-right: 30px;
+  max-width: 100%;
+}
+
+.grid-container {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+  gap: 10px;
+  margin: 0 200px;
+}
+
+.grid-item {
+  position: relative;
+  background-color: #eaeaea;
+  height: 400px;
+}
```

Comment:
- **[OPTIONAL]** Each card should use display flex. It is a project requirement 👇🏽 

> Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.